### DISCUSSION TITLE
Mode 9 only unloads one side at a time now?

### DISCUSSION THREAD

**Discussion by MtBakerScum** [Courseplay/courseplay#7295] [Q&A]
Does mode 9 only unload one side of the bunker at a time now? Its been awhile since I last did silage but I feel like I remember it switching up what lane it unloaded every pass.
![20210619145945_1](https://user-images.githubusercontent.com/54657774/122656443-5f637400-d10f-11eb-836d-fa561bb514bb.jpg)

**Comment by MtBakerScum** [comment]
I think I figured it out, is the working width in mode9 the width of the bunker, and not the width of the tool?

**Reply by MtBakerScum** [reply]
Nope thats not it either

**Reply by Tensuko** [reply]
Working width is still the width of your shovel 
Your issue is sure a tiny bit of silage sticking on the left side at the start, wich we take as target. And we always take the most front heap as target.

**Comment by MtBakerScum** [comment]
I ran the left side bunker down a bit as well, but no matter how much
material is removed from the start, each side continued to only unload one
side at a time.

I think the base game bunkers are at fault, I feel like material gets stuck
in the walls and it gets detected. I'm going to try some different bunkers
and see what happens.

The old mode 9 used to sit at a wait point if it detected the truck it was
unloading into had left, this version does not seems to do that anymore. It
just get stuck in a loop with a loaded fork/bucket. Could that wait feature
be brought back?

**Reply by Tensuko** [reply]
Screenshot of your Bunker at the Front + all waypoints (the 3 dots in the HUD, NOT the eye).

**Comment by MtBakerScum** [comment]
Oh and the measurement bar appears behind the loader and not over the
implement, making it difficult to dial in the appropriate width

**Reply by Tensuko** [reply]
That is becouse of the Mode10, where we need it behind, maybe we can change it for Mode9.

**Comment by MtBakerScum** [comment]
I've already replaced the bunker in my save game, if I have issues with
this next bunker and test batch I'll be sure to update this post.

Thank you though
